_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -80662,6 +80671,601 @@
+	pixel_x = 2;
+	pixel_y = 6
+	},
+/turf/open/floor/plating,
+/area/maintenance/starboard/aft)
+"Qti" = (
+/obj/effect/spawner/structure/window/reinforced,
+/turf/open/floor/plating,
+/area/space)
+"Qtj" = (
+/obj/structure/lattice,
+/turf/open/space/basic,
+/area/space)
+"Qtk" = (
+/obj/structure/lattice,
+/turf/open/space/basic,
+/area/space)
+"Qtl" = (
+/obj/structure/disposalpipe/segment,
+/obj/structure/cable/yellow{
+	icon_state = "1-2"
+	},
+/obj/machinery/atmospherics/pipe/simple/supply/hidden,
+/obj/machinery/door/airlock/research/glass{
+	name = "Circuitry Lab";
+	req_access_txt = "47"
+	},
+/turf/open/floor/plasteel/white,
+/area/science/circuit)
+"Qtm" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qtn" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qto" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qtp" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qtq" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qtr" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qts" = (
+/turf/closed/wall,
+/area/science/circuit)
+"Qtt" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 1
+	},
+/turf/open/floor/plasteel,
+/area/science/circuit)
+"Qtu" = (
+/obj/effect/turf_decal/stripes/line{
+	dir = 5
+	},
+/turf/open/floor/plasteel,
+/area/science/circuit)
+"Qtv" = (
+/obj/machinery/airalarm{
+	pixel_y = 32
+	},
+/obj/structure/disposalpipe/trunk{
+	dir = 4
+	},
+/obj/machinery/disposal/bin,
+/turf/open/floor/plasteel/white,
+/area/science/circuit)
+"Qtw" = (
+/obj/item/device/multitool,
+/obj/item/screwdriver,
+/obj/structure/table/reinforced,
+/obj/structure/cable/yellow{
+	icon_state = "4-8"
+	},
+/obj/machinery/light_switch{
+	pixel_y = 32
+	},
+/turf/open/floor/plasteel/white,
+/area/science/circuit)
+"Qtx" = (
+/obj/structure/table/reinforced,
+/obj/structure/cable/yellow{
+	icon_state = "4-8"
+	},
+/obj/machinery/requests_console{
+	department = "Science";
+	departmentType = 2;
+	dir = 2;
+	name = "Science Requests Console";
+	pixel_y = 30;
+	receive_ore_updates = 1
+	},
+/obj/item/device/integrated_electronics/debugger,
+/turf/open/floor/plasteel/white,
+/area/science/circuit)
+"Qty" = (
+/obj/machinery/light{
+	dir = 1
+	},
+/obj/item/stock_parts/cell/super,
+/obj/item/stock_parts/cell/super,
+/obj/item/stack/sheet/metal/fifty,
+/obj/structure/table/reinforced,
+/obj/structure/cable/yellow{
+	icon_state = "4-8"
+	},
+/obj/machinery/computer/security/telescreen{
+	desc = "Used for watching the RD's goons from the safety of your own office.";
+	name = "Research Monitor";
+	network = list("RD");
+	pixel_y = 32
+	},
+/turf/open/floor/plasteel/white,
+/area/science/circuit)
+"Qtz" = (
+/obj/item/screwdriver,
+/obj/structure/table/reinforced,
+/obj/structure/sign/poster/official/random{
+	pixel_y = 32
+	},
+/obj/structure/cable/yellow{
+	icon_state = "4-8"
+	},
+/turf/open/floor/plasteel/white,
+/area/science/circuit)
+"QtA" = (
+/obj/machinery/power/apc{
+	dir = 1;
+	name = "Circuitry lab area power controller";
```

Comment:
wrong name
no area string

Reply:
What is right name?What is area string?

Reply:
Replace "area power controller" with APC
Areastring should be 
areastring = "/area/science/circuit"
It's not quitttttte necessary in this case since the APC is in the same room, however it's good practice to set it in case you have to move things around. It's essentially just telling the APC what room it should power if you put it in a maintenance tunnel with different area.